doc/install.rst

```diff
@@ -0,0 +1,83 @@
+Build Instructions
+==================
+
+Dependencies
+------------
+
+We are using colcon_ as build tool and typically use `ros2 run` to execute our
+applications.  While we are not really depending on any ROS_ packages, this
+means a basic ROS 2 installation is recommended.
+
+We are testing on Ubuntu 20.04 with ROS Foxy.  Other versions may work as well
+but are not officially supported.
+
+
+Get the Source
```

Comment:
This section is basically the same in all the different package documentations.  Would it make sense to have this only at one place in a generic way and link to there?

Reply:
We could but for now I think this is fine.